pkg/tsdb/cloudwatch/cloudwatch.go

```diff
@@ -80,6 +85,11 @@ func newExecutor(im instancemgmt.InstanceManager, cfg *setting.Cfg, sessions Ses
 	}
 
 	e.resourceHandler = httpadapter.New(e.newResourceMux())
+
+	if len(logsTimeout) == 1 && logsTimeout[0] != 0 {
+		e.logsTimeoutDefault = logsTimeout[0]
```

Comment:
I should probably return an error here if len > 1?

Reply:
(If all goes as I thought, we won't need this anymore, but I think logsTimeout didn't need to be a slice. Then you wouldn't have to do this check. Let me know if I missed something!)